frontend/src/routes/FacilityBooking/ViewBooking/index.tsx

```diff
@@ -145,17 +146,17 @@ export default function ViewBooking() {
           if (data.data === '' || data.data === undefined) {
             console.log(data.err)
           } else {
-            openTelegram(data.data)
+            openTelegram(data.data, newTab)
```

Comment:
is this the only place using openTelegram?

Reply:
good catch, found one more and fixed in #8d98c31